src/Mod/Part/App/AppPart.cpp

```diff
@@ -424,6 +425,7 @@ PyMOD_INIT_FUNC(Part)
     Part::Primitive             ::init();
     Part::Box                   ::init();
     Part::Spline                ::init();
+	Part::FeatureDerivedPart    ::init();

```

Comment:
Any reason for the whitespace change ?

Reply:
On my editor there is no difference. may be due to the size of the tabs. I will correct that.